jenkins/jenkins2-ha.yaml

```diff
@@ -1027,6 +1039,55 @@ Resources:
     Properties:
       ThroughputInMibps: !Ref EFSProvisionedThroughputInMibps
       ServiceToken: !GetAtt 'LambdaFunction.Arn'
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
```

Comment:
I assume access is also managed by IAM? So we don't need a resource-based access policy?

Reply:
yes. we don't need a specifiy policy which means that access is controlled by iam only